psalm.xml

```diff
@@ -75,6 +75,12 @@
                 <referencedClass name="Doctrine\DBAL\Platforms\PostgreSQLPlatform" />
             </errorLevel>
         </InvalidClass>
+        <MethodSignatureMismatch>
+            <errorLevel type="suppress">
+                <!-- Psalm is wrong about ReflectionProperty signatures. -->
```

Comment:
Is there a github issue to track this?

Reply:
I haven't researched that yet.

Reply:
I just filed vimeo/psalm#7357

Reply:
Great! I also observed this locally: https://psalm.dev/r/5c7d43694a , but maybe that's because I'm using 8.0 :thinking:

Reply:
Yes, that error goes away if you run Psalm with PHP 8.1.